connector/checkpoint/checkpoint_view.xml

```diff
@@ -48,6 +48,7 @@
                         delete="false" edit="false" version="7.0">
                     <field name="model_id"/>
                     <field name="record"/>
+                    <field name="message"/>
```

Comment:
You need to install a module called _web_responsive_ for saving horizontal space. Do you know it? :laughing: